packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/CameraPermissionsManager.java

```diff
@@ -0,0 +1,120 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camerax;
+
+import android.Manifest;
+import android.Manifest.permission;
+import android.app.Activity;
+import android.content.pm.PackageManager;
+import androidx.annotation.VisibleForTesting;
+import androidx.core.app.ActivityCompat;
+import androidx.core.content.ContextCompat;
+
+final class CameraPermissionsManager {
```

Comment:
This is the same as [CameraPermissions.java ](https://github.com/flutter/plugins/blob/main/packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/CameraPermissions.java) in the old plugin.